pkgs/data/fonts/times-newer-roman/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchzip }:
+
+let
+  version = "unstable-2018-09-11";
+in
+fetchzip {
+  name = "times-newer-roman-${version}";
+
+  url = "https://timesnewerroman.com/assets/TimesNewerRoman.zip";
```

Comment:
Can we upload this to web.archive.org to get a stable URL?

Reply:
@SuperSandro2000 Done!